x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/__mocks__/mock.ts

```diff
@@ -240,3 +241,9 @@ export const mockRules: Rule[] = [
   mockRule('abe6c564-050d-45a5-aaf0-386c37dd1f61'),
   mockRule('63f06f34-c181-4b2d-af35-f2ace572a1ee'),
 ];
+
+export const mockExceptionsList: List = {
```

Comment:
Nit: I think we've been trying to pull from the existing mocks (`x-pack/plugins/lists/common/schemas/response/exception_list_schema.mock.ts`) for consistency. Def not anything to block this from going in, just fyi.

Reply:
A bit more background and history... We have tech debt to eventually move rules and more things from rules out of the `__mocks__` to be of the form:

```ts
file.ts
file.test.ts
file.mock.ts
```

Where file.ts is the file with the code, file.test.ts is the test file, and then file.mock.ts contains collections of mocks for the file.  

We did it with lists and it has been working well. Reasoning is that it follows more of the Kibana core development model where they do it a lot and it was easier for us to find mocks from core so we started following it more from lists development. So when looking for mocks people should try to find them next to the files they represent. If they don't exist add them. If it does exist but doesn't have the type of mock you want, you can add it to the mock file. Mock files can contain multiple mock functions.

Also, it reduces the amount of duplicate mocking as we also have current issues where a lot of mocks are duplicated. I agree it doesn't have to be with this PR, just wanted to give some back story.